src/sentry/static/sentry/app/components/modals/sudoModal.tsx

```diff
@@ -0,0 +1,170 @@
+import {withRouter} from 'react-router';
+import React from 'react';
+import {ModalHeader, ModalBody} from 'react-bootstrap';
+import {WithRouterProps} from 'react-router/lib/withRouter';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import Alert from 'app/components/alert';
+import withApi from 'app/utils/withApi';
+import Button from 'app/components/button';
+import ConfigStore from 'app/stores/configStore';
+import Form from 'app/views/settings/components/forms/form';
+import InputField from 'app/views/settings/components/forms/inputField';
+import {IconFlag} from 'app/icons';
+import TextBlock from 'app/views/settings/components/text/textBlock';
+import U2fContainer from 'app/components/u2f/u2fContainer';
+import space from 'app/styles/space';
+import {Client} from 'app/api';
+
+type Props = WithRouterProps & {
+  api: Client;
+  closeModal: () => void;
+  Header: typeof ModalHeader;
+  Body: typeof ModalBody;
```

Comment:
I think there is a `ModalProps` (or something like that) type for these

Reply:
oh yeah...the ModalRenderProps...lemme update it...thanks!